weave-js/src/components/PagePanelComponents/Home/Browse3/context.tsx

```diff
@@ -449,6 +463,26 @@ export const browse3ContextGen = (
         leaderboardName ? `/${leaderboardName}` : ''
       }${edit ? '?edit=true' : ''}`;
     },
+    compareCallsUri: (
+      entityName: string,
+      projectName: string,
+      callIds: string[]
+    ) => {
+      const params = callIds
+        .map(id => 'call=' + encodeURIComponent(id))
```

Comment:
Curious why you went with IDs over refs?

Reply:
seems like you could do `ref=...` and the ref itself encodes that it is an object or a call. which would unify the model a bit.

It seems a little odd to me that the search params are mutually exclusive

Reply:
Using refs would make the URLs substantially longer and uglier 

e.g. comparing three calls here:
`http://localhost:9000/wandb-designers/signal-maven/weave/compare?call=491f11f4-cccf-4f0b-b447-897254fa1534&call=d1e494c4-737f-49a8-be89-7703fec23b7b&call=2b58c817-607b-48ed-b4ce-4ec4fb816e95`
v.s. with refs it would be:
`http://localhost:9000/wandb-designers/signal-maven/weave/compare?ref=weave%3A%2F%2F%2Fwandb-designers%2Fsignal-maven%2Fcall%2F491f11f4-cccf-4f0b-b447-897254fa1534&ref=weave%3A%2F%2F%2Fwandb-designers%2Fsignal-maven%2Fcall%2Fd1e494c4-737f-49a8-be89-7703fec23b7b&ref=weave%3A%2F%2F%2Fwandb-designers%2Fsignal-maven%2Fcall%2F2b58c817-607b-48ed-b4ce-4ec4fb816e95`